addons/website/static/src/js/editor/widget_link.js

```diff
@@ -26,43 +28,88 @@ weWidgets.LinkTools.include({
     },
     /**
      * Allows the URL input to propose existing website pages.
-     *
+     * Allows the URL input to be filled from document upload.
+     * Automatically creates a link if selected content is a phone number.
+     * 
      * @override
      */
     start: async function () {
         var def = await this._super.apply(this, arguments);
-        wUtils.autocompleteWithPages(this, this.$('input[name="url"]'));
+        this.$urlInput = this.$('#o_link_dialog_url_input');
+        wUtils.autocompleteWithPages(this, this.$urlInput);
         this._adaptPageAnchor();
+        /*
+         * Strips spaces and parenthesis from link text.
+         * If text is a phone number (only composed of digit except for leading +), make the link callable.
+         * e.g +32 4 95 86 85 25 / 04 95 86 85 25 / +32 (0)4 95 86 85 25
+         */
+        const text = this.data.content.replace(/[ )(]/g, '');
+        const numberStartIndex = text.startsWith('+') ? 1 : 0;
+        if (/^\d+$/.test(text.substr(numberStartIndex))) {
```

Comment:
Should this also let `.`, `/` and `-` go through ? `0499/47.55.66`

Reply:
192.168.0.1 could be an IP, 26/04/2021 a date, do we really want to accept it as phone number ? (currently numbers are also interpreted as phone numbers which can already be wrong). With @qsm-odoo  we agreed on the following condition : starts with  a + or is only composed of digits (without points). What do you think ? Let me know as it's only a matter of appreciation and very little time to update it !

Reply:
I don't agree here, let's keep talking in regular PR thread as this is marked as `Outdated`